src/components/Character/Character.jsx

```diff
@@ -55,29 +54,29 @@ export default function Character(props){
 			aria-roledescription="slide"
 			aria-label={`Character ${index + 1} of ${characterData.length}.`}
 			aria-hidden={ hidden }>
-			<article>
-				<h1 id={`character__${id}`}>
-					{ name }
-				</h1>
-				<h2>
-					{ nickname }
-				</h2>
-				<h3>
-					{ actor }
-				</h3>
-				<p>
-					{ body }
-				</p>
-				<img 
-					src={ src } 
-					alt={ description }
-					style={ mask }
-				/>
-
-				{ prevCharName && RENDER.button(prevCharName, -1) }
-				{ nextCharName && RENDER.button(nextCharName, +1) }
-
+			<article className={ s.container }>
+				<div className={ s.content }>
+					<h1 className={ s.name } 
+						id={`character__${id}`}
+					>
+						{ name }
+					</h1>
+					<h2 className={ s.nickname } >
+						{ nickname }
+					</h2>
+					<h3 className={ s.actor } >
+						{ actor }
+					</h3>
+					<p className={ s.body } > 
```

Comment:
Have I already flagged the `<div clasName={s.body}>` and `RENDER.body` for here?

Reply:
Yeah i was struggling to bring over the render util to do it.
Managed it today though, and updated all of this :)